### DISCUSSION TITLE
MenuButton direction

### DISCUSSION THREAD

**Discussion by dbani-dev** [adobe/react-spectrum#3652] [Q&A]
In this example the `direction` prop does not work for the `MenuButton`. 

https://codesandbox.io/s/lingering-field-42l856

How should `useOverlayPosition` be used?

**Comment by snowystinger** [comment]
We have an example with overlay position here https://react-spectrum.adobe.com/react-aria/useOverlayTrigger.html#example
Let us know if that answers your question.
